content/docs/learning-guide/tutorials/environments/create-terrain-from-images/create-a-terrain-ready-level.md

```diff
@@ -0,0 +1,47 @@
+---
+
+## Goal
+
+In this tutorial section, you will create a new level that is ready for authoring terrain.
+
+## Create a new level
+
+1. In **O3DE Editor**, create a new level. (Refer to the [Create a new level](/docs/learning-guide/tutorials/reference/environments/create-a-level) tutorial for more details)
+
+2. Delete the **Grid**, **Ground**, and **Shader Ball** entities underneath the **Atom Default Environment** so that the terrain you will build is fully visible.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/delete-default-entities.png" alt="Delete several default entities." >}}
+
+3. On the **Global Sky** entity, delete the [**HDRi Skybox**](/docs/user-guide/components/reference/atom/hdri-skybox/) component. The default skybox has terrain in the skybox image which will clash with the terrain that you will create below. Instead, we will use the **Sky Atmosphere** component, which provides an empty sky to work with.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/delete-hdri-skybox.png" alt="Delete the HDRi Skybox component." >}}
+
+4. In **Entity Outliner**, select the **Sun** entity.
+
+5. In **Entity Inspector**, choose **Add Component** and add the **Sky Atmosphere** component. By adding it to the **Sun** entity, the sky lighting will automatically align with the sun's orientation in the sky.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/add-sky-atmosphere.png" alt="Add the Sky Atmosphere component." >}}
+
+At this point, the level should appear empty except for a blue sky.
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/terrain-empty-level.png" alt="Illustration of the new empty level with the sky component configured." >}}
+
+## Enable the terrain system
+
+To enable the terrain system, you must add two level components to the **Level** entity. After adding these components, the terrain system will be enabled, but no terrain will be visible yet.
+
+1. In **Entity Outliner**, select the **Level** entity.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/enable-terrain-select.png" alt="Select the Level entity." >}}
+
+2. In **Entity Inspector**, choose **Add Component** and add both the [**Terrain World**](/docs/user-guide/components/reference/terrain/world) and [**Terrain World Renderer**](/docs/user-guide/components/reference/terrain/world-renderer) level components to the **Level** entity.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/enable-terrain-components.png" width="600" alt="Add terrain level components." >}}
+
+There are many properties available in the terrain level components that configure the terrain system settings for the level. For now, leave the settings at the default values. They are easier to tune once a terrain exists in the level.
```

Comment:
How strongly do you feel about this one? I wanted to make each subtopic a standalone piece so that they can be reused in more tutorials. In particular, the "create-a-terrain-ready-level" would come in handy for _any_ other terrain tutorials that get written.

On the other hand, we only have a single terrain tutorial right now, so this is only a hypothetical problem. We could add the links now, and remove them if/when chunks get reused. Opinions?

Reply:
I don't feel too strongly about it because the links appear in the ToC, resolving.